app/controllers/health_monitor/health_controller.rb

```diff
@@ -10,10 +10,9 @@ class HealthController < ActionController::Base
     def check
       res = HealthMonitor.check(request: request)
 
-      unless HealthMonitor.configuration.environmet_variables.nil?
```

Comment:
`HealthMonitor.configuration.environmet_variables` can be `nil?` indeed?, so perhaps the check should be restored or rewritten as `||= {}`?

In order to accommodate to the existing style, could you change:
- `merge({ 'time' => Time.now.to_s(:db) })` 
to:
- `merge(time: Time.now.to_s(:db))`
?